src/content_delegate.js

```diff
@@ -242,50 +221,67 @@ ContentDelegate.prototype.handleDocketDisplayPage = async function () {
   const radioDateInputs = [...document.getElementsByTagName('input')].filter(
     input => input.name === 'date_from' && input.type === 'radio'
   );
-  if (radioDateInputs.length > 1) {
-    return;
-  };
+  if (radioDateInputs.length > 1) { return; };
+
+  this.recap.getAvailabilityForDocket(
+    this.court,
+    this.pacer_case_id,
+    (result) => {
+      if (result.count === 0) {
+        console.warn('RECAP: Zero results found for docket lookup.');
+      } else if (result.count > 1) {
+        console.error(
+          `RECAP: More than one result found for docket lookup. Found ${result.count}`
+        );
+      } else {
+        const first_result = result.results[0];
+        const mainDiv = document.querySelector('div#cmecfMainContent');
+        mainDiv.insertBefore(recapButton(this.court, this.pacer_case_id), mainDiv.childNodes[0]);
+      }
+    }
+  );
 
   // if you've already uploaded the page, return
-  if (history.state && history.state.uploaded) {
-    return;
-  }
+  if (history.state && history.state.uploaded) { return; }
 
   // check if appellate
-  let isAppellate = PACER.isAppellateCourt(this.court);
+  // let isAppellate = PACER.isAppellateCourt(this.court);
 
   // if the content_delegate didn't pull the case Id on initialization,
   // check the page for a lead case dktrpt url.
   const tabStorage = await getItemsFromStorage(this.tabId)
   const pacerCaseId = this.pacer_case_id ? this.pacer_case_id : tabStorage.caseId;
 
-  if (!pacerCaseId) {
-    // If we don't have any pacerCaseId punt.
-    return;
-  }
+  // If we don't have any pacerCaseId at this point, punt.
+  if (!pacerCaseId) { return; }
 
   const options = await getItemsFromStorage('options');
 
   if (options['recap_enabled']) {
-    let callback = $.proxy(function (ok) {
+    let callback = (ok) => {
       if (ok) {
+        const banner = document.querySelector('.recap-alert-banner');
```

Comment:
Back in the day, `recap_enabled` allowed you to completely turn off RECAP, but nowadays it only means that uploads are turned off. I think putting this behind the `recap_enabled` flag is a mistake then, because even if I have uploads off, I should be able to create an alert.